[
  "[ERROR] Instance count for \u0027Measure.group.population.criteria\u0027 is 0, which is not within the specified cardinality of 1..1 (at Measure.group[0].population[0])"
]